python/tests/test_logistic_regression.py

```diff
@@ -2092,3 +2099,58 @@ def test_quick_double_precision(
         reg_factors=reg_factors,
         float32_inputs=float32_inputs,
     )
+
+
+@pytest.mark.slow
+def test_sparse_int64() -> None:
+
+    gpu_number = 1
+    data_shape = (int(1e7), 30)
```

Comment:
Will this give only 300 million nonzero?   Thats below the max int limit to be tested?

Reply:
Revised to use larger dataset and SparseDataGen.